tests/storages_tests/rdb_tests/test_with_server.py

```diff
@@ -115,3 +134,26 @@ def test_multiprocess(storage_url: str) -> None:
     assert len(trials) == n_workers * 20
 
     _check_trials(trials)
+
+
+@pytest.mark.parametrize("n_jobs", [1, 4])
+def test_multiprocess_with_progbar(storage_url: str, n_jobs: int) -> None:
+    n_workers = 8
+    study_name = _STUDY_NAME
+    optuna.create_study(storage=storage_url, study_name=study_name)
+    with Pool(n_workers) as pool:
```

Comment:
```suggestion

    with Pool(n_workers) as pool:
```

Reply:
I don't think that blank line is necessary because existing ```test_multiprocess``` does not have blank line before ```with Pool(n_workers) as pool:```.